safaribooks.py

```diff
@@ -1121,4 +1124,4 @@ def create_epub(self):
 
     SafariBooks(args_parsed)
     # Hint: do you want to download more then one book once, initialized more than one instance of `SafariBooks`...
-    sys.exit(0)
+    return
```

Comment:
**Suggestion:** The 'return' statement at the end of the script is not necessary. Python scripts end naturally when they run out of code to execute.
```suggestion
    SafariBooks(args_parsed)
    # Hint: do you want to download more then one book once, initialized more than one instance of `SafariBooks`...
```